CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
 # [Versions](https://mui.com/versions/)
 
+## v6.0.0-alpha.1
+
+<!-- generated comparing v6.0.0-alpha.0..next -->
+
+_Apr 3, 2024_
+
+A big thanks to the 16 contributors who made this release possible. Here are some highlights ✨:
+
+- 🔥 Converted 3 Material UI components to support static CSS extraction. The progress is at 36%!
```

Comment:
It might be a super dumb question, but it doesn't hurt to confirm: is "static CSS extraction" the same as using Pigment CSS? If yes, I think we should say it :) Like: 

> Converted 3 more Material UI components to use Pigment CSS. Current progress is 36%.